platform/windows/display_server_windows.cpp

```diff
@@ -4579,9 +4579,22 @@ DisplayServerWindows::DisplayServerWindows(const String &p_rendering_driver, Win
 	// Init context and rendering device
 #if defined(GLES3_ENABLED)
 
-	if (rendering_driver == "opengl3") {
-		int gl_version = detect_wgl_version();
-		if (gl_version < 30003) {
+	bool fallback = GLOBAL_GET("rendering/gl_compatibility/fallback_to_angle");
+	if (fallback && (rendering_driver == "opengl3")) {
```

Comment:
Don't we still need a path to check for gl_version support when not using the fallback? This seems like it will fail for devices that don't have the fallback enabled and don't support the required version

Reply:
> This seems like it will fail for devices that don't have the fallback enabled and don't support the required version

Isn't it what is supposed to do in such case?